generate_licenses.sh

```diff
@@ -2,91 +2,16 @@
 
 set -euxo pipefail
 
-: ${GENERATOR_HOME?"need the location of the license-generator"}
+: ${GENERATOR_HOME:?"missing environment variable GENERATOR_HOME with the license generator location"}
 
-case "$(basename $(pwd))" in
-  *-redhat)
-    product="true"
-    ;;
-  *)
-    product="false"
-    ;;
-esac
+dir=$(pwd)
 
+cd $GENERATOR_HOME
+mvn clean package -Dexample.project.dir="$dir" -Dexternal.license.service="$LICENSE_SERVICE"
```

Comment:
That will work without the external service as well, right ?

Reply:
Yes: https://github.com/thorntail-examples/booster-license-generator/blob/master/src/main/java/io/thorntail/examples/license/generator/ThorntailGeneratorProperties.java#L34